static/js/components/email/EmailCompositionType.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import React from 'react';
+import { RadioButton, RadioButtonGroup } from 'material-ui/RadioButton';
+
+import {
+  ONE_TIME_EMAIL,
+  EMAIL_CAMPAIGN
+} from './constants';
+
+export default class EmailCompositionType extends React.Component {
```

Comment:
We'll need to add more UI for rendering other options for these emails, so I think it would make sense to call this something else, maybe `AutomaticEmailOptions` or similar.